generator/projectmanager.h

```diff
@@ -50,19 +50,12 @@ struct ProjectInfo {
         :   name(std::move(name)), source_path(std::move(source_path)), revision(std::move(rev))  {}
 };
 
-
 struct ProjectManager {
-    explicit ProjectManager(std::string outputPrefix, std::string _dataPath)
-            : outputPrefix(std::move(outputPrefix)) , dataPath(std::move(_dataPath))
-    {
-        if (dataPath.empty())
-            dataPath = "../data";
-    }
+    ProjectManager::ProjectManager(std::string outputPrefix, std::string _dataPath);
```

Comment:
Why did you removed explicit?

Please keep it: `{"/foo", "/bar"}`  is not a ProjectManager

Reply:
That was by accident, when moving things around (did go a bit back and forth about how to structure this change).